### PR TITLE
[Modular] Anti-Gravity Boots, but real.

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Updates and fixes the first feature of #6071, which adds anti-gravity boots for ~~cargo to wear and use and abuse.~~ The QM, as his second special special boy item - now having two, like the CE. /tg/ style traitor objectives haven't been made for these, so they won't count as such if those ever get reimplemented for whatever reason.
Weh.

## Why It's Good For The Game
Helps speed up the manual labor of cargo either through pushing off objects and walls or using a jetpack / thruster implant - the latter is VERY fun In my opinion. Of course it's a high value item and VERY conspicuous, so only the bravest of souls will use it illicitly to it's full potential. Or the QM will just beg the HOP for EVA. I forsee the latter, honestly.

## Changelog
:cl:
add: The QM now has a pair of anti-gravity boots in their locker. Speed out, man!
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	desc = "Anti-gravity boots, for those who want to live weightlessly. Control Click to toggle anti-gravity functions."
+	name = "anti-gravity boots"
+	icon_state = "walkboots" //Haha funny reused sprite
+	var/enabled_antigravity = FALSE
+
+/obj/item/clothing/shoes/antigrav_boots/equipped(mob/user, slot)
+	. = ..()
+	if(slot == ITEM_SLOT_FEET)
+		if(enabled_antigravity)
+			user.AddElement(/datum/element/forced_gravity, 0)
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm]:
```suggestion
	if(!slot == ITEM_SLOT_FEET)
		return
	if(enabled_antigravity)
		user.AddElement(/datum/element/forced_gravity, 0)
```

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by tf-4** [comment]
Would be better as an antag item imo.

**Comment by unit0016** [comment]
> Would be better as an antag item imo.

It would basically necessitate also buying a hardsuit or breaking into EVA if the boots were traitor items, which would completely balls up the experience for security because near every traitor would be super duper fast, meaning they'd have to set up barricades in hallways and just etcetera etcetera. **I am open to making them high-value items in the QM's locker (just the one pair), like the CE's magboots and blueprints are, however.** (Plus anti-gravity is already obtainable if miners hand off geyser chems to chemistry (but they never do.))

**Comment by tf-4** [comment]
Won't making them printable have the exact same issues?

I think making them a unique QM item is also a good solution

File: modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	if(!slot == ITEM_SLOT_FEET)
+		return
+	if(enabled_antigravity)
+		user.AddElement(/datum/element/forced_gravity, 0)
+
+/obj/item/clothing/shoes/antigrav_boots/dropped(mob/user)
+	. = ..()
+	user.RemoveElement(/datum/element/forced_gravity, 0)
+
+/obj/item/clothing/shoes/antigrav_boots/CtrlClick(mob/living/user)
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm:21]: could move this to right click?

Comment (unit0016) [reply modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm:21]: They already have to remove the boots to change it, might as well make it clankier.

**Pull request closed without merging**